master/usage/configuration/mtu.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Configuring MTU
+---
+
+Depending on the environment Calico is being deployed into it may be
+helpful or even necessary to configure the MTU of the TAP (or veth) that is
+attached to each workload and the tunnel devices if IP-in-IP is enabled.
+
+### Selecting MTU size
+
+Typically the MTU for your workload interfaces should match the optimum MTU
```

Comment:
Optimum MTU sounds feels like an odd concept to me, isn't it more that for optimum transmission the workload interfaces should match the network MTU.